src/quadratic.jl

```diff
@@ -36,9 +36,9 @@ function coefficients(q::Quadratic, N, d)
     symm, sym, symp =  symbol(string("cm_",d)), symbol(string("c_",d)), symbol(string("cp_",d))
     symfx = symbol(string("fx_",d))
     quote
-        $symm = convert(TIndex, .5 * ($symfx - .5)^2)
-        $sym  = convert(TIndex, .75 - $symfx^2)
-        $symp = convert(TIndex, .5 * ($symfx + .5)^2)
+        $symm = convert(TIndex, 1//2 * ($symfx - 1//2)^2)
```

Comment:
Is specifying these literals as `Rational`s better (than `Float64`s) in some general sense, or does it just happen to work better with `Rational` indices/coefficients? Might there be other cases where we want/need to specify these literals in yet another type to make it work nicely?